{
    "type": "bundle",
    "id": "bundle--8edf4223-078b-4262-810c-1cc762968f62",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--454d35ae-b944-4698-b940-0f8c393a73a5",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2025-02-21T00:20:07.237522Z",
            "modified": "2025-02-21T00:20:07.237522Z",
            "name": "CVE-2025-0868",
            "description": "A vulnerability, that could result in Remote Code Execution (RCE), has been found in DocsGPT. Due to improper parsing of JSON data using eval() an unauthorized attacker could send arbitrary Python code to be executed via /api/remote endpoint.\n\nThis issue affects DocsGPT: from 0.8.1 through 0.12.0.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2025-0868"
                }
            ]
        }
    ]
}